models/eprop_iaf_psc_delta.h

```diff
@@ -35,72 +35,122 @@
 
 namespace nest
 {
-
+// Disable clang-formatting for documentation due to over-wide table.
+// clang-format off
 /* BeginUserDocs: neuron, e-prop plasticity, integrate-and-fire, current-based
 
 Short description
 +++++++++++++++++
 
-Current-based leaky integrate-and-fire neuron model with delta-shaped
-postsynaptic currents for e-prop plasticity
+Leaky integrate-and-fire model with delta-shaped input currents
+for e-prop plasticity
 
 Description
 +++++++++++
 
-``eprop_iaf_psc_delta`` is an implementation of a leaky integrate-and-fire model
-where the potential jumps on each spike arrival used for eligibility propagation
-(e-prop) plasticity.
+``iaf_psc_delta`` is a leaky integrate-and-fire neuron model with
+
+* a hard threshold,
+* a fixed refractory period,
+* Dirac delta (:math:`\delta`)-shaped synaptic input currents.
+* support for eligibility propagation (e-prop).
 
 It is the standard ``iaf_psc_delta`` model endowed with e-prop plasticity.
```

Comment:
```suggestion
``iaf_psc_delta`` is a leaky integrate-and-fire neuron model with

* a hard threshold,
* a fixed refractory period,
* Dirac delta (:math:`\delta`)-shaped synaptic input currents.


The `eprop_iaf_psc_delta` is the standard ``iaf_psc_delta`` model endowed with e-prop plasticity.
```

This first part is not entirely clear to me. Is the above suggestion what you really meant?

Reply:
Yes, sorry!